src/content/language/records.md

```diff
@@ -124,9 +124,17 @@ check out the page on [Patterns][pattern].
 
 ## Record types
 
-There is no type declaration for individual record types. Records are structurally
-typed based on the types of their fields. A record's _shape_ (the set of its fields,
-the fields' types, and their names, if any) uniquely determines the type of a record. 
+Records in Dart are structurally typed based on their fields' types, names, and count. 
+While you can't declare a unique type for a record shape, you can use `typedef` for readability and reuse. 
+Learn more in the [Records and typedefs](https://dart.dev/language/records#records-and-typedefs) 
```

Comment:
To learn more, see [Records and typedefs](https://dart.dev/language/records#records-and-typedefs).